clients/src/main/java/org/apache/kafka/common/requests/AbstractResponse.java

```diff
@@ -106,189 +107,189 @@ public static AbstractResponse parseResponse(ByteBuffer buffer, RequestHeader re
                 requestHeader.correlationId(), responseHeader.correlationId());
         }
 
-        return AbstractResponse.parseResponse(apiKey, buffer, apiVersion);
+        return AbstractResponse.parseResponse(apiKey, new ByteBufferAccessor(buffer), apiVersion);
     }
 
-    public static AbstractResponse parseResponse(ApiKeys apiKey, ByteBuffer responseBuffer, short version) {
+    public static AbstractResponse parseResponse(ApiKeys apiKey, ByteBufferAccessor readable, short version) {
```

Comment:
If you agree with my suggestion, you can change this signature to `parseResponse(ApiKeys, Readable, short)`.

Reply:
Done.